tests/unit/src/main/scala/tests/BaseLspSuite.scala

```diff
@@ -223,3 +225,10 @@ abstract class BaseLspSuite(
     }
   }
 }
+
+object ServerLivenessTestData {
```

Comment:
Maybe let's put it in `ServerLivenessMonitorSuite` file?

Reply:
Can't because of the project structure.